tests/testthat/_snaps/step-call-pivot_wider.md

```diff
@@ -0,0 +1,17 @@
+# names_glue affects output names & auto-converts data.table to lazy_dt
+
+    Code
+      show_query(step)
+    Output
+      setnames(dcast(`_DT5`, formula = "... ~ x + y", value.var = c("a", 
+      "b"))[, .(a_X_1, a_Y_2, b_X_1, b_Y_2)], old = c("a_X_1", "a_Y_2", 
+      "b_X_1", "b_Y_2"), new = c("X1_a", "Y2_a", "X1_b", "Y2_b"))
+
+# can sort column names
+
+    Code
+      show_query(step)
+    Output
+      setcolorder(dcast(DT, formula = "... ~ chr", value.var = "int")[, 
+          .(Wed, Tue, Mon)], c("Mon", "Tue", "Wed"))
```

Comment:
Could we not generate:

```R
cast(DT, formula = "... ~ chr", value.var = "int")[,  (Mon, Tue, Wed)]
```

Or does that make the renaming code too complicated?

Reply:
All set.

FYI - there are two cases if `names_sort = TRUE`.

Case 1: `length(id_cols) == 0` (like the snapshot above)
The translation now reads like you recommended (only uses the "select" step, no `setcolorder()` call).

Case 2: `length(id_cols) > 0`
There is no "select" step necessary in this case, so it still uses `setcolorder()` for names sorting.

If you'd rather have Case 2 use "select" as well let me know.